OktaAspNetCoreMvc/Controllers/AccountController.cs

```diff
@@ -26,12 +38,36 @@ public IActionResult Logout()
             }
 
             return RedirectToAction("Index", "Home");
-        }
-
+        }    
+     
         [Authorize]
         public IActionResult Claims()
         {
             return View();
         }
+
+        [Authorize]
+        public async System.Threading.Tasks.Task<IActionResult> Me()
+        {
+           var oktaClient = new OktaClient(
```

Comment:
Let's do this part in Startup.cs. That way, we can keep all the Configuration stuff in one place, and also add `IOktaClient` to the dependency injection system so it can be injected into any controller :)